### PR TITLE
Support Statamic v4

### PR DESCRIPTION
**Pull request by jimblue**
Fix for #4

### CODE REVIEW

**Comment by jimblue** [comment]
I still need to test it, please wait before create a release

**Comment by jimblue** [comment]
Hi @michaelr0 !
So, the first commit is for compatibility with Statamic V4... I've just test it an it's working! 🚀 
The second commit improve the PHP quality but is totally optional.
I let you decide what you want to include in the new release.
Cheers 😉

File: tests/TestCase.php [hunk 1]
```diff
@@ -9,11 +9,10 @@
 use Statamic\Providers\StatamicServiceProvider;
 use Statamic\Statamic;
 
-class TestCase extends OrchestraTestCase
+final class TestCase extends OrchestraTestCase
```
Comment (michaelr0) [inline tests/TestCase.php:12]: Please remove final from this test case

**Comment by michaelr0** [comment]
Thanks Jim, please re-add the doc block return types.

please revert the yoda style comparison `null == $var` to `is_null` or `$var == null`

please address the styleci issues.

Seeing as this drops support for prior versions,  can you investigate if it is possible to use a version check to allow this to function for prior versions of Statamic, if Statamic is still offering updates to those versions. Thanks again.

File: tests/TestCase.php (hunk 1 above)
Comment (jimblue) [reply tests/TestCase.php]: done

**Comment by jimblue** [comment]
> Seeing as this drops support for prior versions, can you investigate if it is possible to use a version check to allow this to function for prior versions of Statamic, if Statamic is still offering updates to those versions. Thanks again.

No idea how to do that but anyway this will just add some dead code and more maintenance.

It would be simpler and better for this project to just create a new release and with a breaking change  reference  in the release note.

**Comment by jimblue** [comment]
@michaelr0 I've address all **styleci** issues !
Some issues looked a bit random (the 2 spaces asked for @param...) so I've place them in a fixup.
Let me know if you to want to squash it or fix it a different way.

**Comment by michaelr0** [comment]
Thanks @jimblue

**Pull request merged**